Bluepill-runner/BluepillRunnerTests/BPReportCollectorTests.m

```diff
@@ -80,11 +80,23 @@ - (void)testCollectReportsFromPath {
     NSXMLElement *root = testsuitesNodes[0];
     NSString *got = [[root attributeForName:@"tests"] stringValue];
 
-    XCTAssertTrue([got isEqualToString:@"25"], @"test count is wrong, wanted 25, got %@", got);
+    XCTAssertTrue([got isEqualToString:@"26"], @"test count is wrong, wanted 25, got %@", got);
     got = [[root attributeForName:@"errors"] stringValue];
     XCTAssertTrue([got isEqualToString:@"2"], @"error count is wrong, wanted 2, got %@", got);
     got = [[root attributeForName:@"failures"] stringValue];
-    XCTAssertTrue([got isEqualToString:@"2"], @"failure count is wrong, wanted 2, got %@", got);
+    XCTAssertTrue([got isEqualToString:@"3"], @"failure count is wrong, wanted 2, got %@", got);
```

Comment:
Same as above

Reply:
Made sure we don't screw this up again ;-)